appstudio-controller/controllers/appstudio.redhat.com/promotionrun_controller.go

```diff
@@ -396,12 +406,12 @@ func checkForExistingActivePromotions(ctx context.Context, reconciledPromotionRu
 	return nil
 }
 
-func locateTargetManualBinding(ctx context.Context, promotionRun appstudioshared.PromotionRun, k8sClient client.Client) (appstudioshared.SnapshotEnvironmentBinding, error) {
+func locateOrCreateTargetManualBinding(ctx context.Context, promotionRun appstudioshared.PromotionRun, k8sClient client.Client, log logr.Logger) (appstudioshared.SnapshotEnvironmentBinding, error) {
```

Comment:
This file imports:

```go
"github.com/go-logr/logr"
"sigs.k8s.io/controller-runtime/pkg/log"
```

There is overshadowing of the `log` package with the `log` variable. And this is happening in a lot of places in our code, so at some point we have to address it.

Reply:
Fixed.